cmake-modules/AddAzureSDKforC.cmake

```diff
@@ -17,7 +17,7 @@
 include(FetchContent)
 FetchContent_Declare(azuresdkforc
     GIT_REPOSITORY      https://github.com/Azure/azure-sdk-for-c.git
-    GIT_TAG             1.0.0-preview.6)
+    GIT_TAG             1.0.0)
```

Comment:
@vhvb1989 - this highlights the need for https://github.com/Azure/azure-sdk-for-c/issues/1311 to auto update versions. We don't have the 1.0.0-preview.6 tag.